spec/models/zip_code_spec.rb

```diff
@@ -1,7 +1,72 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require './spec/support/test_helper'
 
 RSpec.describe ZipCode, type: :model do
-  it { should validate_presence_of(:d_codigo) }
+  include Helper
+
+  context 'validations tests' do
+    it { should validate_presence_of(:d_codigo) }
+  end
+
+  context 'scopes tests' do
+    let!(:zip_code) { FactoryBot.create(:zip_code) }
+
+    it 'should return a zip_code' do
+      expect(ZipCode.count).to eq(1)
+    end
+
+    it 'search for a wrong zip_code gives empty results' do
+      create_cp(%w[57300 21360 81920])
+      cp = '64000'
```

Comment:
can you please name the variable in a more meaningful way?, `cp` is actually in spanish

Reply:
Done!